examples/python/download_dashboard_pdf.py

```diff
@@ -8,28 +21,18 @@
 import looker_sdk
 from looker_sdk import models
 
-import sdk_exceptions
-
-sdk = looker_sdk.init31("../../looker.ini")
+sdk = looker_sdk.init40("../looker.ini")
```

Comment:
oops, down one level too many
```suggestion
sdk = looker_sdk.init40("../../looker.ini")
```